selfdrive/car/subaru/carcontroller.py

```diff
@@ -41,6 +47,49 @@ def update(self, c, CS, frame, actuators, pcm_cancel_cmd, visual_alert, left_lin
 
       self.apply_steer_last = apply_steer
 
+    # *** stop and go ***
+
+    throttle_cmd = False
+
+    if CS.CP.carFingerprint in PREGLOBAL_CARS:
```

Comment:
CC has CC.cruiseControl.cancel which is used for pcm_cancel_cmd in carcontorllers. Do you mean I should add cancel and resume fields to CruiseState in cereal and use those instead of CS.cruise_state

I reviewed car.capnp and I think I can also use CS.brakeHoldActive instead of CS.cruise_state = 3 and maybe also CS.cruiseState.standstill instead CS.out.standstill

Reply:
would CS.cruiseState.leadVisible (CS.car_follow) and CS.cruiseState.leadDist (CS.close_distance) also make sense in cereal or would it be better to keep them as local variables?

Reply:
Is there a smaller version of this PR you can come up with that just simply sends the gas resume message to take off from a light, without all the other checks?